docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -179,7 +179,7 @@ This cannot be combined with `deny-unannotated-endpoints`, as `deny` takes the e
 
 `quarkus.security.deny-unannotated-members=true|false`::
 - if set to true, the access will be denied to all CDI methods
-and Jakarta REST endpoints that do not have security annotations but are defined in classes that contain methods with
+and JAX-RS endpoints that do not have security annotations but are defined in classes that contain methods with
```

Comment:
@MichalMaler I think it is fine, as `Jakarta` is an implementation detail, Guillaume, let me know please if it is of concern, I'm not sure we need to use `Jakarta` in such contexts

Reply:
OK, will keep it as it is if @gsmet agrees with you on this. Dont want to interfere with anybody elses intention. But these changes should be hand over to docs team since it affects our work (in two repositories at the same time).

Reply:
No it is not fine. I did a PR specifically to change that so I would appreciate if it wasn't lost.

Reply:
It will be back as I said earlier (if this will be merged), dont worry @gsmet .
I just thought that we should know about these renaming changes, thats all.